In [25]:
import pandas as pd
import numpy as np
import matplotlib.pyplot as plt

In [26]:
parlamentarios_info = pd.read_csv('participacion/parlamentarios_info_general.csv')
descripcion_2023_1 = pd.read_csv('participacion/participacion_descripcion_2023_1.csv')
descripcion_2023_2 = pd.read_csv('participacion/participacion_descripcion_2023_2.csv')
descripcion_2024 = pd.read_csv('participacion/participacion_descripcion_2024.csv')

In [27]:
combined_df = pd.concat([descripcion_2023_1, descripcion_2023_2, descripcion_2024], ignore_index=True)


total_rows = combined_df.shape[0]
unique_values = combined_df["DESCRIPCION_DEBATE"].nunique()
print(f"'DESCRIPCION_DEBATE' distintos: {unique_values} en {total_rows} filas")


'DESCRIPCION_DEBATE' distintos: 5932 en 68161 filas


Hay que separar los 5932, ver si se va a hacer con keywords 